cEpiabm/src/sweeps/household_sweep.cpp

```diff
@@ -78,7 +78,7 @@ namespace epiabm
 
         double infectiousness = Covidsim::CalcHouseInf(infector, timestep);
         double susceptibility = Covidsim::CalcHouseSusc(infector, infectee, timestep);
-        double foi = infectiousness * susceptibility;
+        double foi = infectiousness * susceptibility * 0.5;
 
         if (static_cast<double>(std::rand() % 1000000) / static_cast<double>(1000000) < foi)
```

Comment:
I am confused about the factor of 1/2 here - this isn't currently consistent with pyEpiabm but should we add it there?

And why the use of a million in L83?

Reply:
I think covid-sim had that factor, could be wrong. Used million to generate random float to 6dp, in the future I imagine this will be handled by some random number classes elsewhere

Reply:
Ooh okay - will have a look as the python doesn't have this so one backend needs to be changed!

Reply:
@NicholasFan235 have been going through the CovidSim code on this and I really can't find this factor of 1/2 - have you got a reference for it?

Reply:
You're right, 0.5 factor just appears in a later commit, probably from testing